devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1617,10 +1663,10 @@ bool HumanStateProvider::impl::applyRpcCommand()
         iDynTree::Twist baseVel;
         baseVel.zero();
 
-
-        // setting to zero all the selected joints
         for (auto const& jointZeroIdx: jointZeroIndices) {
-            jointPos.setVal(jointZeroIdx, 0);
+
+            jointPos.setVal(jointZeroIdx, jointConfigurationSolution.getVal(jointZeroIdx));
+            
         }
```

Comment:
As in https://github.com/robotology/human-dynamics-estimation/pull/341/files#r1138933864.

Reply:
As per https://github.com/robotology/human-dynamics-estimation/pull/341#discussion_r1140014766, this changes are ok.